system-tests/test/before_browser_launch_spec.ts

```diff
@@ -25,7 +25,8 @@ describe('e2e before:browser:launch', () => {
   })
 
   systemTests.it('can add extensions', {
-    browser: '!webkit', // TODO(webkit): fix+unskip, or skip and add a test that this fails with WebKit
+    // as of Chrome 137, --load-extension does not work in chrome branded. In order to test this, we need to use the chrome-for-testing browser or chromium.
+    browser: ['electron', 'chrome-for-testing', 'firefox'], // TODO(webkit): fix+unskip, or skip and add a test that this fails with WebKit
```

Comment:
Should this just be not webkit or chrome?
```suggestion
    // as of Chrome 137, --load-extension does not work in chrome branded browsers, so exclude chrome
    // TODO(webkit): fix+unskip, or skip and add a test that this fails with WebKit
    browser: ['!webkit', '!chrome'],
```

Reply:
I think so. I figured it would be better to specify what specifically you want tested vs trying to infer what is being tested via what isn't being tested

Reply:
but we do this elsewhere too so I don't have a strong opinion

Reply:
Yeah, I just saw two different patterns in this PR and thought we should settle on one. I would just use the exclude pattern for this one.

Reply:
agreed. Got this cleaned up in 2bf85aa7e3